index.html

```diff
@@ -1,12 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="./assets/css/all.min.css">
+    <link rel="stylesheet" href="./assets/css/main.css" />
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather</title>
 </head>
-<body>
-    
+<body> 
+  <div class="site-wrapper">
+    <section class="first-section">
+      <div class="first-child-fs">
+        <i class="fa-solid fa-magnifying-glass"></i>
+      </div>
+    </section>
+    <section class="second-section">
+      <div class="first-child-ss">city</div>
+      <div class="second-child-ss">date</div>
+    </section>
+    <section class="third-section">
+      <div class="first-child-ts">
+      <i class="fa-solid fa-sun"></i>
+      </div>
+      <div class="second-child-ts">25&deg;C</div>
+      <p class="third-child-ts">Description</p>
+    </section>
+      <section class="fourth-section">
+        <div class="first-child-fs">
+          <i class="fa-solid fa-wind"></i>
+        </div>
+        <div class="second-child-fs">
+          <i class="fa-solid fa-droplet"></i>
+        </div>
+      </section>
+      <section class="fifth-section">
+        <div class="first-child-fifth-s">
+          <div>
```

Comment:
this div can be removed. in this part, there are a lot of divs that can be removed try to clean this part and remove extra elements.